### ISSUE TITLE
Allow user to set conserved_dna and ani threshold

### ISSUE THREAD

**Issue by natir** [oschwengers/referenceseeker#2]
Hello,

I have a set of Nanopore and Pacbio bacteria reads. I perform assemblies with [wtdbg2 or redbean](https://github.com/ruanjue/wtdbg2) I run referenceseeker mash found some potential genome but I think the ANI threshold was to high for my uncorrected assemblies. It could be useful if user can set these two thresholds.

In fact I want the most nearest available reference genome not necessary the real one.

For example I with dataset [ERR2531096](https://www.ebi.ac.uk/ena/data/view/ERR2531096) assembled with wtdbg2:
```
wtdbg2 -t 16 -g 2.1m -x sq -i ERR2531096.fasta -fo ERR2531096_wtdbg2
wtpoa-cns -t 16 -i ERR2531096_wtdbg2.ctg.lay.gz -fo ERR2531096_wtdbg2.fasta
```

In run referenceseeker against prebuild bacteria database in verbose mode I get this output:
```
ReferenceSeeker v1.2.0
Options, parameters and arguments:
	use bundled binaries: False
	db path: /scratch/bioinf/users/pmarijon/yacrd-and-fpa-upstream-tools-for-lr-genome-assembly/referenceseeker/bacteria
	genome path: /scratch/bioinf/users/pmarijon/yacrd-and-fpa-upstream-tools-for-lr-genome-assembly/assembly/ERR2531096_pb.raw.wtdbg2.fasta
	tmp path: /tmp/tmpmm8f0tg0
	unfiltered: False
	# CRG: 10
	# threads: 16

Estimate genome distances...
	screened 63 potential reference genome(s)
	reduce to best 10 hits...

Compute ANIs...

#ID	ANI	Con. DNA	Mash Distance	Taxonomy ID	Assembly Status	Organism
```

**Comment by oschwengers** [comment]
Hi,
> In fact I want the most nearest available reference genome not necessary the real one

This is exactly what ReferenceSeeker was developed for. What do you mean by "the real one"?

> Allow user to set conserved_dna and ani threshold

That's quite a nice idea! I'll implement it asap. Meanwhile, you could try the `--unfiltered` option in order to completely bypass the `ANI` and `conserved DNA` filters.

**Comment by natir** [comment]
> What do you mean by "the real one"?

Sorry I'm not very clear I mean the exact reference of this assembly.

> Meanwhile, you could try the --unfiltered

Oh sorry I didn't understand cleary the help message for this option my bad.

**Comment by oschwengers** [comment]
@natir I just released v1.3 https://github.com/oschwengers/referenceseeker/releases/tag/v1.3
The BioConda update is also available... Have fun and let me know if there's more to enhance.
